x-pack/test_serverless/api_integration/test_suites/search/common_configs/config.group1.ts

```diff
@@ -25,6 +25,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       require.resolve('../../common/search_oss'),
       require.resolve('../../common/search_xpack'),
       require.resolve('../../common/core'),
+      require.resolve('../../common/reporting'),
```

Comment:
also this for `api_integration/test_suites/security/common_configs/config.group1.ts`
also this for `api_integration/test_suites/observability/common_configs/config.group1.ts`

Reply:
ah good catch thank you!!